src/boot/bootctl-status.c

```diff
@@ -130,6 +177,33 @@ static int print_efi_option(uint16_t id, int *n_printed, bool in_order) {
 
         efi_tilt_backslashes(path);
 
+        if (arg_json_format_flags != JSON_FORMAT_OFF) {
+                _cleanup_(json_variant_unrefp) JsonVariant *efi_obj = NULL;
+                _cleanup_free_ char *status_buffer = NULL;
+                _cleanup_free_ char *partition_buffer = NULL;
+                char uuid_str[SD_ID128_STRING_MAX];
+
+                asprintf(&status_buffer, "%sactive%s", active ? "" : "in", in_order ? ", boot-order" : "");
```

Comment:
this needs to be split into two separate json fields instead of a log-like string